www/pages/search_data.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+permalink: /howto/search_data
+---
+<!--- IMPORTANT: This permlink is referenced from InputApp -->
```

Comment:
is it?

Reply:
No...I did not add the comment!

Reply:
we have automatic tests in inputapp that check that all pages that inputapp references exists. so this comment should be there only if this is such permlink page. see https://github.com/lutraconsulting/input/blob/c99b56c317f43dbb9b5ad69cd331d4d7c78cbbdd/app/test/testlinks.h#L50